ansible/vagrant-webservers.yml

```diff
@@ -39,7 +39,7 @@
         isic_smtp_port: "1025"
         # S3 development is mocked with minio
         isic_upload_bucket_name: "test-upload-bucket"
-        isic_upload_role_arn: "fake-arn"
+        isic_upload_role_arn: "fake-arn-that-is-at-least-20-characters"
```

Comment:
Don't change this, but do put a comment saying AWS requires 20 chars and MinIO doesn't.